src/test/java/org/wso2/carbon/connector/integration/test/CeridianDayforce/CeridianDayforceConnectorIntegrationTest.java

```diff
@@ -0,0 +1,1095 @@
+        Assert.assertEquals(eiRestResponse.getBody().getString("Data"), (apiRestResponse.getBody().getString("Data")));
+    }
+
+    @Test(enabled = true, groups = {"wso2.ei"}, description = "getEmployeeUSTaxStatuses Test Case")
+    public void getEmployeeUSTaxStatuses() throws Exception {
+        RestResponse<JSONObject> eiRestResponse =
+                sendJsonRestRequest(getProxyServiceURLHttp("getEmployeeUSTaxStatuses"), "POST",
+                        eiRequestHeadersMap, "getEmployeeUSTaxStatuses.json");
+
+        String apiEndPoint = connectorProperties.getProperty("apiUrl") + "/" + connectorProperties
+                .getProperty("apiVersion") + "/Employees/42199/USTaxStatuses";
+        RestResponse<JSONObject> apiRestResponse = sendJsonRestRequestHTTPS(apiEndPoint, "GET", apiRequestHeadersMap,
+                null, null);
+
+        Assert.assertEquals(eiRestResponse.getHttpStatusCode(), 200);
+        Assert.assertEquals(apiRestResponse.getHttpStatusCode(), 200);
+        Assert.assertEquals(eiRestResponse.getBody().getString("Data"), (apiRestResponse.getBody().getString("Data")));
+    }
+
+    @Test(enabled = true, groups = {"wso2.ei"}, description = "getEmployeeClockDeviceGroups Test Case")
+    public void getEmployeeClockDeviceGroups() throws Exception {
+        RestResponse<JSONObject> eiRestResponse =
+                sendJsonRestRequest(getProxyServiceURLHttp("getEmployeeClockDeviceGroups"), "POST",
+                        eiRequestHeadersMap, "getEmployeeClockDeviceGroups.json");
+
+        String apiEndPoint = connectorProperties.getProperty("apiUrl") + "/" + connectorProperties
+                .getProperty("apiVersion") + "/Employees/63499/ClockDeviceGroups";
+        RestResponse<JSONObject> apiRestResponse = sendJsonRestRequestHTTPS(apiEndPoint, "GET", apiRequestHeadersMap,
+                null, null);
+
+        Assert.assertEquals(eiRestResponse.getHttpStatusCode(), 200);
+        Assert.assertEquals(apiRestResponse.getHttpStatusCode(), 200);
+        Assert.assertEquals(eiRestResponse.getBody().getString("Data"), (apiRestResponse.getBody().getString("Data")));
+    }
+
+    @Test(enabled = true, groups = {"wso2.ei"}, description = "getEmployeeCompensationSummary Test Case")
+    public void getEmployeeCompensationSummary() throws Exception {
+        RestResponse<JSONObject> eiRestResponse =
+                sendJsonRestRequest(getProxyServiceURLHttp("getEmployeeCompensationSummary"), "POST",
+                        eiRequestHeadersMap, "getEmployeeCompensationSummary.json");
+
+        String apiEndPoint = connectorProperties.getProperty("apiUrl") + "/" + connectorProperties
+                .getProperty("apiVersion") + "/Employees/42199/CompensationSummary";
+        RestResponse<JSONObject> apiRestResponse = sendJsonRestRequestHTTPS(apiEndPoint, "GET", apiRequestHeadersMap,
+                null, null);
+
+        Assert.assertEquals(eiRestResponse.getHttpStatusCode(), 200);
```

Comment:
These three lines have been duplicated several times. Can't we do something about it?